person.rb

```diff
@@ -0,0 +1,27 @@
+require './person'
+
+class Person
+  attr_accessor :name, :age
+  attr_reader :id
+
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @id = Random.rand(1..1000)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
```

Comment:
- [ ] As requested by the previous reviewer, From the project requirement, you should Make sure that this class has a method `correct_name` implemented but now you're missing the `correct_name` method which should simply return the name attribute. :blush: